src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -5472,20 +5472,22 @@ public class PipelineAst : PipelineBaseAst
         /// </summary>
         /// <param name="extent">The extent of the pipeline.</param>
         /// <param name="pipelineElements">The collection of commands representing the pipeline.</param>
+        /// <param name="backgroundProcess">Indicates that this pipeline should be run in the background</param>
         /// <exception cref="PSArgumentNullException">
         /// If <paramref name="extent"/> is null.
         /// </exception>
         /// <exception cref="PSArgumentException">
         /// If <paramref name="pipelineElements"/> is null or is an empty collection.
         /// </exception>
-        public PipelineAst(IScriptExtent extent, IEnumerable<CommandBaseAst> pipelineElements)
+        public PipelineAst(IScriptExtent extent, IEnumerable<CommandBaseAst> pipelineElements, bool backgroundProcess)
             : base(extent)
```

Comment:
Should we maybe inherit another class from `PipelineAst` instead? Something like `BackgroundPipelineAst`.